buildSrc/src/main/kotlin/buildsrc/convention/android-library.gradle.kts

```diff
@@ -42,6 +49,20 @@ android {
     }
 }
 
+dependencies {
+    testImplementation("junit:junit:${Deps.Versions.junit4}")
+    androidTestImplementation("androidx.test.espresso:espresso-core:${Deps.Versions.androidxEspresso}") {
+        exclude("com.android.support:support-annotations")
+    }
+
+    androidTestImplementation("androidx.test:rules:${Deps.Versions.androidxTestRules}")
+    androidTestImplementation("androidx.test:runner:${Deps.Versions.androidxTestRunner}")
+    androidTestImplementation("androidx.test.ext:junit-ktx:${Deps.Versions.androidxTestExtJunit}")
+
+    androidTestImplementation(kotlin("test"))
+    androidTestImplementation(kotlin("test-junit"))
+}
+
```

Comment:
📝I thought it would be better to make the test configs common, so I moved them here.